operator/v1/types_console.go

```diff
@@ -146,6 +146,22 @@ type ConsoleCustomization struct {
 	// +listMapKey=id
 	// +optional
 	Perspectives []Perspective `json:"perspectives"`
+	// consoleURL is an URL to be used as the base console address.
+	// This field is intended for clusters without ingress capability,
+	// where access to routes is not possible.
+	// Invalid value will prevent a console rollout.
+	// +kubebuilder:validation:Optional
+	// +optional
+	// +kubebuilder:validation:XValidation:rule="self.startsWith('https://')",message="console url scheme must be https"
+	ConsoleURL string `json:"consoleURL,omitempty"`
+	// downloadsURL is an URL to be used as the address to download client binaries.
+	// This field is intended for clusters without ingress capability,
+	// where access to routes is not possible.
+	// Invalid value will prevent a console rollout.
+	// +kubebuilder:validation:Optional
+	// +optional
```

Comment:
You only need one optional tag, drop the kubebuilder version

Reply:
Done.